CONTRIBUTING.md

```diff
@@ -41,15 +47,14 @@ Creating a pull request is fairly simple, just make sure it focuses on a single
 
 We would greatly appreciate the code submitted to be of a consistent style with other code in disnake. This project follows PEP-8 guidelines (mostly) with a column limit of 100 characters.
 
-We use [`nox`](https://nox.thea.codes/en/stable/) for automating development tasks. Run these commands to
-install `nox` and `taskipy` as well as the required dependencies in your environment,
-and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.  
-Make sure you have a virtualenv activated if you don't want it to install the packages globally!
+Run these commands to install [`nox`](https://nox.thea.codes/) and [`pdm`](https://pdm.fming.dev/), as well as the required dependencies in your environment, and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.
 ```
-pip install nox taskipy
-task setup_env
+pip install nox pdm
+pdm setup_env
```

Comment:
Do we want to use the `pdm <script>` shortcut (like right here) or the full `pdm run <script>` (like below)? Either way, it should be consistent.

Reply:
I think its fine to go either way here. Do you have a preference?

Reply:
`pdm run`, probably. It'll avoid any possible conflicts in the future, but you can still use the shorthand if you know about it :p

Reply:
```suggestion
pdm run setup_env
```